tests/meltano/cli/test_initialize.py

```diff
@@ -53,3 +53,26 @@ def test_init(self, cli_runner, tmp_path_factory, pushd):
         meltano_yml = project.root_dir("meltano.yml").read_text()
         assert "send_anonymous_usage_stats: false" in meltano_yml
         assert "project_id:" in meltano_yml
+
+    def test_init_existing_empty_directory(self, cli_runner, tmp_path_factory, pushd):
```

Comment:
@ReubenFrankel do you think it makes sense to add a test for the case when the user passes an _absolute_ path. The updated CLI docs say the argument must be a relative directory, so what happens when the user passes an absolute path?

Reply:
The path can be absolute or relative - I'll update the docs and add a test case. :smile:

Reply:
https://github.com/meltano/meltano/pull/6814/commits/acf4bd22ce3f7d1c665800d84b02b75783ef7577

Reply:
https://github.com/meltano/meltano/pull/6814/commits/12d18ee2a1e58b5358f4fc83168b28da812ed02f